package.json

```diff
@@ -26,7 +26,7 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "deploy-prod": "react-scripts build && firebase deploy",
+    "deploy-prod": "react-scripts build && firebase deploy --only hosting",
```

Comment:
what does "--only hosting" do?

Reply:
i believe it only releases the web app. ive yet to test tho

Reply:
the reason this is important is bc if we dont specify, it automatically deploys everything (db, functions, etc.)